redpanda/README.md

```diff
@@ -0,0 +1,134 @@
+# Agent Check: Redpanda
+
+## Overview
+
+This Datadog-[Redpanda][1] integration collects key metrics from Redpanda by default. It can also be configured to add additional metric groups based on specific user needs.
+
+Redpanda is a Kafka API compatible streaming platform for mission-critical workloads.
+
+## Setup
+
+### Installation
+
+First, [download and launch the Datadog Agent][9].
+
+Then, manually install the Redpanda check. [Instructions vary depending on the environment][10].
+
+> Current Redpanda integration version: `1.0.0`
+
+#### Host
+
+Run `datadog-agent integration install -t datadog-redpanda==<INTEGRATION_VERSION>`.
+
+#### Containerized
+
+The best way to use this integration with the Docker Agent is to build the Agent with this integration installed. Use the following Dockerfile to build an updated version of the Agent:
+
+```dockerfile
+FROM gcr.io/datadoghq/agent:latest
+
+ARG INTEGRATION_VERSION=1.0.0
+
+RUN agent integration install -r -t datadog-redpanda==${INTEGRATION_VERSION}
+```
+
+Build the image and push it to your private Docker registry.
+
+Then, upgrade the Datadog Agent container image. If the Helm chart is used, modify the `agents.image` section in the `values.yaml` to replace the default agent image:
+
+```yaml
+agents:
+  enabled: true
+  image:
+    tag: <NEW_TAG>
+    repository: <YOUR_PRIVATE_REPOSITORY>/<AGENT_NAME>
+```
+
+Use the new `values.yaml` to upgrade the Agent:
+
+```shell
+helm upgrade -f values.yaml <RELEASE_NAME> datadog/datadog
+```
+
+### Configuration
+
+#### Host
+
+##### Metric collection
+
+1. Edit the `redpanda.d/conf.yaml` file in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your Redpanda performance data. See the sample [redpanda.d/conf.yaml.example][3] for all available configuration options.
+
+2. [Restart the Agent][4].
+
+##### Log collection
+
+_Available for Agent versions >6.0_
+
+1. Collecting logs is disabled by default in the Datadog Agent, enable it in your `datadog.yaml` file:
+
+   ```yaml
+   logs_enabled: true
+   ```
+
+2. Add this configuration block to your `redpanda.d/conf.yaml` file to start collecting your Redpanda logs:
+
+   ```yaml
+    logs:
+    - type: journald
+      source: redpanda
+    ```
+
+#### Containerized
+
+##### Metric collection
+
+For containerized environments, after the Redpanda check is integrated in the Datadog Agent image, Autodiscovery is configured by default.
+
+Thus, metrics are automatically collected to Datadog's server.
+
+See the [Autodiscovery Integration Templates][2] for the complete guidance.
+
+##### Log collection
+
+_Available for Agent versions >6.0_
+
+Collecting logs is disabled by default in the Datadog Agent. To enable it, see [Kubernetes log collection documentation][9].
```

Comment:
By default, log collection is disabled in the Datadog Agent. Log collection is available for Agent v6.0+.